app/models/post.rb

```diff
@@ -1,8 +1,13 @@
 class Post < ApplicationRecord
+  after_save :update_post_counter
+
   belongs_to :author, class_name: 'User'
   has_many :comments
   has_many :likes
-  after_save :update_post_counter
+
+  validates :title, presence: true, length: { maximum: 250 }
+  validates :comments_counter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
+  validates :likes_counter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
```

Comment:
- [ ] I was unable to create posts. After adding these validations, you still need to update the `create` action in the Post controller to account for `likes_counter` and `comments_counter`. Kindly update the `create` action so that likes and comments are set to a value (like 0) before `@post.save`.